packages/dd-trace/src/serverless.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+const log = require('./log')
+
+function maybeStartServerlessMiniAgent () {
+  const isGCPFunction = process.env.K_SERVICE !== undefined || process.env.FUNCTION_NAME !== undefined
+  const rustBinaryPath = process.env.DD_MINI_AGENT_PATH
+
+  if (!isGCPFunction) {
+    return
+  }
+  if (!rustBinaryPath) {
+    log.error('Serverless Mini Agent did not start. Please provide a DD_MINI_AGENT_PATH environment variable.')
+    return
+  }
+
+  try {
+    const { spawn } = require('child_process')
+
+    const miniAgentProcess = spawn(rustBinaryPath)
+
+    miniAgentProcess.stdout.on('data', (data) => {
+      log.debug(data.toString())
+    })
+    miniAgentProcess.on('close', (code) => {
+      log.debug(`Mini Agent exited with code ${code}`)
```

Comment:
Should this be an error (or at least a warning) when it exits with a non-zero code?